File: pinyougou-web-manager/src/main/webapp/js/service/orderService.js
app.service('orderService',function ($http) {
    
    this.search=function (page,rows,searchEntity) {
        //page,rows,$scope.searchEntity)
        return $http.post("../order/search.do?page="+page+"&rows="+rows,searchEntity);
    }


    this.updateStatus=function (status, ids) {
        return $http.get("../order/updateStatus.do?status="+status+"&ids="+ids);
    }
})

File: pinyougou-interface/src/main/java/cn/itcast/core/service/OrderService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.item.Item;
import cn.itcast.core.pojo.order.Order;
import cn.itcast.core.pojo.order.OrderItem;
import entity.PageResult;
import vo.OrderVo;

import java.util.List;

public interface OrderService {
    void add(Order order);

    PageResult search(Integer page, Integer rows, Order order);

    void updateStatus(String status, Long[] ids);

    /**
     *   通过商家ID 查询 订单表的订单总数(即销量)
     */
    Integer selectCountBySellerId(String sellerId);

    List<OrderVo> findOrderListByUsername(String name);

    List<OrderItem> findOrderItem(String orderId);

    Item findSpecByItemId(Long itemId);
}
